### PR TITLE
feature/showEmail

### PR DESCRIPTION
**Pull request by VirtualReality1**
Added Email to User profile, minor Layout Changes

### CODE REVIEW

File: resources/views/profiles/show.blade.php [hunk 1]
```diff
@@ -12,6 +12,11 @@
 
                 <p class="text-secondary uppercase text-xl font-semibold text-center">{{ $user-> firstName}} {{ $user-> lastName}}</p>
                 <p class="text-secondary text-center mt-2"> @ {{ $user ->username }}</p>
+                @if($user->username == Auth::user()->username ||$user->showMail == "1" ? $user->email : "" )
```
Comment (P4inty) [inline resources/views/profiles/show.blade.php:15]: Kommentar bei #18 beachten, hier ist das gleiche Problem

File: resources/views/profiles/show.blade.php [hunk 2]
```diff
@@ -23,25 +28,25 @@ class="text-center bg-secondary text-white rounded-full font-medium py-1 my-1">F
...
-                        <p class="text-secondary font-semibold  mb-4 ml-8">Themen</p>
-                        <div class="flex text-center " style="display: table-cell">
-                        @foreach($user->tags as $tag)
-                            <p href="#"
-                               class="text-center flex-col text-sm text-secondary border-secondary font-medium uppercase border-2 my-1 px-1 mx-2">{{$tag->name}}</p>
-                        @endforeach
+                        <div class="flex text-center " style="display: table; margin: auto">
+                            <p class="text-secondary font-semibold text-center mb-4 ml-16 ">Themen</p>
+                            @foreach($user->tags as $tag)
+                                 <p href="#"
```
Comment (P4inty) [inline resources/views/profiles/show.blade.php:47]: #18 beachten

File: resources/views/profiles/show.blade.php [hunk 3]
```diff
@@ -44,8 +44,7 @@ class="bg-secondary text-primary rounded-full font-medium uppercase py-1 my-1 mx
                         <div class="flex text-center " style="display: table; margin: auto">
                             <p class="text-secondary font-semibold text-center mb-4 ml-16 ">Themen</p>
                             @foreach($user->tags as $tag)
-                                 <p href="#"
-                                     class="text-sm text-secondary border-secondary font-medium uppercase border-2 my-1 px-1 mx-2 ml-16" >{{$tag->name}}</p>
+                                 <p class="text-sm text-secondary border-secondary font-medium uppercase border-2 my-1 px-1 mx-2 ml-16" >{{$tag->name}}</p>
```
Comment (VirtualReality1) [inline resources/views/profiles/show.blade.php:47]: @P4inty Adjustments made

File: resources/views/profiles/show.blade.php [hunk 4]
```diff
@@ -12,6 +12,11 @@
 
                 <p class="text-secondary uppercase text-xl font-semibold text-center">{{ $user-> firstName}} {{ $user-> lastName}}</p>
                 <p class="text-secondary text-center mt-2"> @ {{ $user ->username }}</p>
+                @if($user == Auth::user() || $user->showMail ? $user->email :"")
+
```
Comment (P4inty) [inline resources/views/profiles/show.blade.php:16]: ` @if($user == Auth::user() || $user->showMail)`

File: resources/views/profiles/show.blade.php [hunk 5]
```diff
@@ -12,7 +12,7 @@
 
                 <p class="text-secondary uppercase text-xl font-semibold text-center">{{ $user-> firstName}} {{ $user-> lastName}}</p>
                 <p class="text-secondary text-center mt-2"> @ {{ $user ->username }}</p>
-                @if($user == Auth::user() || $user->showMail ? $user->email :"")
+                @if($user == Auth::user() || $user->showMail)
```
Comment (VirtualReality1) [inline resources/views/profiles/show.blade.php]: Done

**Pull request merged**